src/utils/dataTestIds.ts

```diff
@@ -43,9 +43,19 @@ export const dataTestId = {
     nationalIdentityNumberField: 'national-identity-number-field',
     personAdmin: {
       cristinId: (id: string) => `cristin-id-${id}`,
-      nin: (id: string) => `nin-${id}`,
+      employments: (id?: string) => (id ? `employments-${id}` : 'employments'),
+      firstName: 'first-name',
+      institution: 'institution',
+      lastName: 'last-name',
       name: (id: string) => `name-${id}`,
-      employments: (id: string) => `employments-${id}`,
+      nin: (id?: string) => (id ? `nin-${id}` : 'nin'),
+      personSearchField: 'person-search-field',
+      position: 'position',
+      positionPercent: 'positionPercent',
```

Comment:
bare for å beholde naming convention
```suggestion
      positionPercent: 'position-percent',
```